src/Adapter/MSTest.CoreAdapter/Resources/Resource.resx

```diff
@@ -317,4 +317,7 @@ Error: {1}</value>
   <data name="UTA_TestMethodExpectedParameters" xml:space="preserve">
     <value>Only data driven test methods can have parameters. Did you intend to use [DataRow] or [DynamicData]?</value>
   </data>
+  <data name="Execution_Test_Cancelled" xml:space="preserve">
```

Comment:
Please xlf file changes associated with this change

Reply:
Run command "build.cmd -uxlf". You would have to commit the xlf files changes post running the script.